### ISSUE TITLE
Malware

### ISSUE THREAD

**Issue by ghanki** [windowtoolbox/powershell-windows-toolbox#31]

**Comment by RandonDude** [comment]
Hi, ich habe einen Wiederherstellungspunkt genutzt. Das dürfte helfen oder?

**Comment by ghanki** [comment]
> Hi, ich habe einen Wiederherstellungspunkt genutzt. Das dürfte helfen oder? 

I can't be sure. Did you do the system restore point before installing the tool, or not?

**Comment by RandonDude** [comment]
Yes of course. The restore point was a few days before the script ran. Reinstalling Windows would take me a day. I hope this helped and I'll keep you posted.

**Comment by ghanki** [comment]
> Yes of course. The restore point was a few days before the script ran. Reinstalling Windows would take me a day. I hope this helped and I'll keep you posted.

Great to hear that the restore point was before the installation, the script on the site is some sort of advertisement bot.

**Comment by Kurotaku-sama** [comment]
> Yes of course. The restore point was a few days before the script ran. Reinstalling Windows would take me a day. I hope this helped and I'll keep you posted.

Zur Sicherheit deinstallier und lösch alle Dateireste von Chrome basierten Browsern wie Chrome, Brave und Edge und installier die Browser neu, könnte sein das ein teil des Codes sich da reingefressen hat und dadurch nicht von Systemwiederherstellungen betroffen und nachwievor infiziert ist.

**Comment by ghanki** [comment]
> > Yes of course. The restore point was a few days before the script ran. Reinstalling Windows would take me a day. I hope this helped and I'll keep you posted.
> 
> 
> 
> Zur Sicherheit deinstallier und lösch alle Dateireste von Chrome basierten Browsern wie Chrome, Brave und Edge und installier die Browser neu, könnte sein das ein teil des Codes sich da reingefressen hat und dadurch nicht von Systemwiederherstellungen betroffen und nachwievor infiziert ist.

The code apparently writes itself into system files, which makes the deletion of the browser not so effective. I recommend to just reinstall windows completely and having a backup of documents or other saved files.
